tests/integration/test.basics.js

```diff
@@ -1106,5 +1106,21 @@ adapters.forEach(function (adapter) {
         });
       });
     }
+
+    if (typeof process !== 'undefined' && !process.browser) {
```

Comment:
TBH I don't remember why I said that. It was a dumb thing to say. 😅 

That said I can modify your PR to remove the "node-only" line and then we'll test it and then :+1: from me.